projects/025 - Watkinson/template-sql/objectives-1-5.template.sql

```diff
@@ -65,6 +67,8 @@ SELECT * FROM #Temp;
 
 --> EXECUTE query-get-covid-vaccines.sql
 
+--> EXECUTE query-received-flu-vaccine.sql date-from:2019-07-01 date-to:2020-06-30
+
 --> EXECUTE query-get-flu-vaccine-eligible.sql
```

Comment:
has this been reviewed already? do you want me to have a look at this as well?

Reply:
Yes please - though you spotted that the > should be >= in the mentioned file so I assume you did actually look at this in the end. Is that right?

Reply:
No that file has already been reviewed by George.